crates/typos-cli/src/bin/typos-cli/report.rs

```diff
@@ -280,6 +286,148 @@ impl Report for PrintJson {
     }
 }
 
+#[derive(Debug)]
+pub(crate) struct PrintSarif {
+    results: Mutex<RefCell<Vec<sarif::Result>>>,
+    error: Mutex<RefCell<Vec<String>>>,
+}
+
+impl Default for PrintSarif {
+    fn default() -> Self {
+        Self {
+            results: Mutex::new(RefCell::new(Vec::new())),
+            error: Mutex::new(RefCell::new(Vec::new())),
+        }
+    }
+}
+
+fn error_to_io_error(error: Box<dyn std::error::Error>) -> std::io::Error {
```

Comment:
Please move a helper like this to down below the `impl`.  At least for me, `impl`s should come right after their type

Reply:
Ok